test/topology/conftest.py

```diff
@@ -175,6 +177,7 @@ async def manager_internal(event_loop, request):
     yield manager_int
     await manager_int.stop()  # Stop client session and close driver after last test
 
+@pytest.mark.asyncio
```

Comment:
This is only used for marking test cases, it serves no purpose for other functions.
https://pytest-asyncio.readthedocs.io/en/latest/reference/markers.html

"A coroutine or async generator with this marker will be treated as a **test function** by pytest."